backend/createGoTerms.sh.in

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Parses the go terms out of an input file
+
+set -e
```

Comment:
You might also want to consider setting the `pipefail` shell option. This way, when `genontology.org` is down or they decided to change their URL or something, this script fails and you get notified instead of failing silently.


_[Original comment](https://github.ugent.be/unipept/unipept/pull/581#discussion_r4630) by svsegher on Tue Jun 14 2016 at 15:53._